### ISSUE TITLE
 Possibility of deactivation polling via flow

### ISSUE THREAD

**Issue by thomasja27** [irritanterik/homey-tesla.com#35]
Hi,
Is it possible to create an option to enable / disable polling via a flow? This is to reduce the power consumption of the car when it is parked for an extended period of time. 
As it is now, the car never sleeps when polling is on.

Best Regards
Thomas Jakobsen

**Comment by PieterLubbers** [comment]
I would also like this possibility, another suggestion would be a separate option: "Polling time while parked". Setting this to 30 minutes ensures that the car will have gone to sleep (if not used). After it sleeps, regular polling can be used again since they will time-out.

**Comment by irritanterik** [comment]
I added both requests in test version 2.1.1 available here: https://homey.app/nl-nl/app/com.tesla/Tesla/test/ 

I did not had the oppertunity to test this version lacking access to a real Tesla driving device or time to arrange this.

**Comment by PieterLubbers** [comment]
Will start testing right away, thanks for the quick response!

**Comment by fschaal** [comment]
@irritanterik, Thank you for adding this and thank you for creating this app. I just installed the test version. I cloud not find the  "Polling time while parked" in the new version. Is this setting turned on by default?

I will let you know if I find any strange behavior. If you need any future help with testing or development let me know! I'm willing to help. Driving a model 3 on a daily basis.

**Comment by fschaal** [comment]
As far as I can tell the model 3 now goes to sleep as expected. The setting for disabling polling works also very nice since you can now integrate it into your "Set home in sleep mode" where I then disable polling until the home changes back to awake mode.

**Comment by robkor** [comment]
Is this _setting for disabling polling_ the actioncard where you can choose out of 4 settings to turn on or off? 

![image](https://user-images.githubusercontent.com/20802916/72226310-21643700-3590-11ea-8c32-a483066fc3b2.png)

**Comment by irritanterik** [comment]
Yes it is. But be aware, changing multiple values from one flow within 5 seconds may result in some troubles. I'm working on a fix for this.

**Comment by robkor** [comment]
Perfect. I will put in a delay of 15 seconds on the second action to prevent trouble.

**Comment by derBlitzen** [comment]
Hi there, 

I also would like to disable/enable polling via flows but can't figure out how to do that to save my life. Could someone help me out?

Cheers,
-Peter
